Automation/Automation/Tests/Wishlist.cs

```diff
@@ -0,0 +1,61 @@
+﻿using FluentAssertions;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Automation.Helpers;
+
+/*[assembly: Parallelize(Workers = 4,
+    Scope = ExecutionScope.MethodLevel)]*/
+namespace Automation.Tests
+{
+    [TestClass]
+    public class Wishlist : BaseTest
+    {
+        [TestInitialize]
+        public override void Before()
+        {
+            base.Before();
+
+            Pages.HomePage.GoToLoginPage();
+
+            Pages.LoginPage.InsertCredentials();
+
+            Pages.LoginPage.SubmitLogin();
+        }
+
+        [TestMethod]
+        public void AddtoWishlistSimpleProductTest()
+        {
+            Pages.HomePage.GoToHomepage();
+
+            Pages.HomePage.GoToPrivateSales();
+
+            var productAddedtoWishlist = Pages.PrivateSalesPage.GetBriefcaseName();
+
+            Pages.PrivateSalesPage.AddBriefcaseToWishlist();
+
+            Pages.PrivateSalesPage.IsAdded(productAddedtoWishlist);
```

Comment:
this test does not have an assert. it will always pass

Reply:
and the name of the method does not really indicate what is added? rename it. eg.g "IsProductAddedTo..."